### PR TITLE
Clarifications and fixes for the README

### PR DESCRIPTION
**Pull request by Barthelemy**
- Clarify the scope
- Fix the URLs
- Clarify the website link
- Remove duplicated information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-AliceO2
-=======
+ALICE O2 software 
+=================
 
-Alice O2 project software. Simulation, reconstruction and common software for
-the ALICE experiment at CERN based on ALFA and the FairRoot software.
+### Scope
+The ALICE O2 software repository contains the framework, as well as the detector specific, code for the reconstruction, calibration and simulation for the ALICE experiment at CERN for Run 3 and 4. It also encompasses the commonalities such as the data format, and the global algorithms like the global tracking. 
+Other repositories in AliceO2Group contain a number of large common modules, per instance the Monitoring or the Configuration.
```
Comment (sawenzel) [inline README.md]: per instance == for instance?

Comment (Barthelemy) [inline README.md]: you are correct

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-Alice O2 project software. Simulation, reconstruction and common software for
-the ALICE experiment at CERN based on ALFA and the FairRoot software.
+### Scope
+The ALICE O2 software repository contains the framework, as well as the detector specific, code for the reconstruction, calibration and simulation for the ALICE experiment at CERN for Run 3 and 4. It also encompasses the commonalities such as the data format, and the global algorithms like the global tracking. 
+Other repositories in AliceO2Group contain a number of large common modules, for instance for Monitoring or Configuration.
 
-### Documentation
-The documentation single entry point is [here](https://alice-o2.web.cern.ch/).
+### Website
+The main entry point for O2 information is [here](https://alice-o2.web.cern.ch/).
```
Comment (matthiasrichter) [inline README.md]: At this point the public web site (http) should be cited (again the old story)

File: README.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
 
-AliceO2
-=======
+ALICE O2 software
```
Comment (matthiasrichter) [inline README.md]: there is an extra blank at the end, but we do not have a tool the check for formatting guideline "Never put trailing whitespace at the end of a line"

File: README.md [hunk 4]
```diff
@@ -1,12 +1,13 @@
 
-AliceO2
-=======
+ALICE O2 software 
+=================
 
-Alice O2 project software. Simulation, reconstruction and common software for
-the ALICE experiment at CERN based on ALFA and the FairRoot software.
+### Scope
+The ALICE O2 software repository contains the framework, as well as the detector specific, code for the reconstruction, calibration and simulation for the ALICE experiment at CERN for Run 3 and 4. It also encompasses the commonalities such as the data format, and the global algorithms like the global tracking.
```
Comment (matthiasrichter) [inline README.md]: "Never put trailing whitespace at the end of a line"

**Comment by Barthelemy** [comment]
Hi Matthias, 
Sorry that you did not get a chance to comment before it was merged. Concerning the URL, surprisingly the home page can be accessed with https whereas the copyright page can't. IT is still looking into this... As a consequence I think that the URL can stay as is it is right now. 
You are correct for the extra blank spaces. l will change the settings of my IDE.

**Pull request merged**